src/main/resources/ddl.sql

```diff
@@ -0,0 +1,94 @@
+/*! SET storage_engine=INNODB */;
+
+
+drop table if exists dwolla_hpp_requests;
+create table dwolla_hpp_requests (
```

Comment:
TBD with the second flow, but you might not need it (for Adyen, it was required to store on disk the mapping payment <-> user because Adyen's notification won't provide the latter).